README.md

```diff
@@ -13,28 +16,28 @@ This means Powershell must be used for all configuration.
 * The entire virtual drive is added to the system as a single large volume
 * You need at least 
   * 1 SSD and 1 HDD to run cached storage / Simple resiliency
-  * 2 SSD and 2 HDD to run cached storage / Mirror resiliency / 
-  * 1 SSD and 2 HDD to run cached storage / Simple resiliency / striped storage (sum of HDD space)
+  * 2 SSD and 2 HDDs to run cached storage / Mirror resiliency / 
+  * 1 SSD and 2 HDDs to run cached storage / Simple resiliency / striped storage (sum of HDD space)
 
 ![Simple](./images_folder/simple.png) ![Mirrored](./images_folder/mirror-simple.png) ![Mirrored and Striped](./images_folder/mirror-stripe.png)
 
 # Scripts 
-## new-storage-space.ps1
+## New-TieredStorageSpace.ps1
 Creates a tiered storage pool and allocates all the disk space to a single drive
```

Comment:
Can we add a line here or at the top that tells us how to run the script with the external configuration file?